.gitignore

```diff
@@ -1,5 +1,5 @@
 target/
-shared/Cargo.lock
+Cargo.lock
```

Comment:
This was intentionally included. Originally shared was not a direct member of a workspace, and sometimes it would get a Cargo.lock because I opened it in vscode. It's no longer needed to be in this file, but Cargo.lock should be included.

The primary purpose of this crate is an executable, and the goal of Cargo.lock in git is to ensure reproducible builds of executables.

Reply:
Done.